md/bpm-api.md

```diff
@@ -481,6 +481,18 @@ Retrieve humanTask objects that match the specified filters.
   `GET`
 * **Data Params**  
   [Standard search parameters](rest-api-overview.md#resource_search) are available.  
+  
+  * o: only parameters available in HumanTaskInstanceSearchDescriptor.java and in the result are accepted: 
```

Comment:
furthermore, the information from this class is true when you call Engine Java API directly but not REST API where not all parameter are supported...
Checkout [AbstractHumanTaskDatastore](https://github.com/bonitasoft/bonita-web/blob/master/server/src/main/java/org/bonitasoft/web/rest/server/datastore/bpm/flownode/AbstractHumanTaskDatastore.java#L76) for the exhaustive list.